Cartfile

```diff
@@ -1,8 +1,8 @@
-github "LoopKit/LoopKit" "dev"
-github "LoopKit/CGMBLEKit" "dev"
+github "LoopKit/LoopKit" "merge-from-tidepool"
+github "LoopKit/CGMBLEKit" "merge-from-tidepool"
 github "i-schuetz/SwiftCharts" == 0.6.5
-github "LoopKit/dexcom-share-client-swift" "dev"
-github "LoopKit/G4ShareSpy" "dev"
-github "ps2/rileylink_ios" "dev"
+github "LoopKit/dexcom-share-client-swift" "merge-from-tidepool"
+github "LoopKit/G4ShareSpy" "merge-from-tidepool"
+github "ps2/rileylink_ios" "merge-from-tidepool"
```

Comment:
[nit] why not dev here?

Reply:
This is an artifact of me making the tidepool -> diy merge available for DIY testers to try out using the normal build method (non-workspace) that most DIYers use.

Tidepool doesn't use this file in its builds at all, so I'd just treat this as noise for now.